index.html

```diff
@@ -28,3 +29,53 @@
         </div>
     </div>
 </div>
+
+{% raw  %}
+<script type="text/javascript">
+    //assume Mustache is loaded
+    //assume Meetup is loaded
+    
+    // Modify from _includes/meetup_cards.html 
```

Comment:
Can we refactor that code so it's reusable without copying and pasting?

Reply:
I moved all the js code to `js/meetup-events.js`. If you want, I can refactor `_includes/meetup_cards.html` and `_includes/developers_cards.html` so they share the same code file.